shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/slf4j/LoggerHelper.java

```diff
@@ -23,4 +23,8 @@ public static LoggingEventBuilder addFatalMarker(LoggingEventBuilder loggingBuil
         Marker fatal = MarkerFactory.getMarker("FATAL");
         return loggingBuilder.addMarker(fatal);
     }
+
+    public static LoggingEventBuilder buildLogBuilder(Logger logger, Level level) {
+        return logger.atLevel(level).setMessage("");
+    }
```

Comment:
This looks like it is unused and can be deleted.

Reply:
done.